docs/ibm-cloud-rules.md

```diff
@@ -256,6 +257,12 @@ for any resources (paths) that support the <code>If-Match</code> and/or <code>If
 <td>oas3</td>
 </tr>
 <tr>
+<td><a href="#ibm-integer-attributes">ibm-integer-attributes</a></td>
+<td>warn</td>
```

Comment:
Docs should be updated to reflect new severity

```suggestion
<td>error</td>
```

Reply:
fixed